### PR TITLE
Change paths for deployment directory

### PR DESCRIPTION
**Pull request by ShafanaSafwan**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/ei_integrator/manifests/init.pp [hunk 1]
```diff
@@ -82,15 +26,15 @@
     file { "$wso2_path/$product-$product_version/${template}":
       ensure  => file,
       mode    => '0644',
-      content => template("${module_name}/carbon-home/${template}.erb")
+      content => template("/var/lib/jenkins/workspace/EI-6.4.0/configs/modules/${module_name}/templates/carbon-home/${template}.erb")
```
Comment (msmshariq) [inline modules/ei_integrator/manifests/init.pp]: @ThilinaManamgoda @ShafanaSafwan - can we parameterize the path value? It's duplicated below as well.

File: modules/ei_integrator/manifests/init.pp [hunk 2]
```diff
@@ -82,15 +26,15 @@
...
     }
   }
 
   # Copy wso2server.sh to installed directory
-  file { "$wso2_path/$product-$product_version/${start_script_template}":
+    file { "$wso2_path/$product-$product_version/${start_script_template}":
     ensure  => file,
     mode    => '0754',
-    content => template("${module_name}/carbon-home/${start_script_template}.erb")
+    content => template("/var/lib/jenkins/workspace/EI-6.4.0/configs/modules/${module_name}/templates/carbon-home/${start_script_template}.erb")
```
Comment (msmshariq) [inline modules/ei_integrator/manifests/init.pp]: Can we parameterize the path value as suggested above?

File: modules/ei_integrator/manifests/init.pp (hunk 1 above)
Comment (shafanaS) [reply modules/ei_integrator/manifests/init.pp]: Changes done in commit e7465959328e3cd44a2d2d1acc89dd06b5921126

File: modules/ei_integrator/manifests/init.pp (hunk 2 above)
Comment (shafanaS) [reply modules/ei_integrator/manifests/init.pp]: Changes done in commit e7465959328e3cd44a2d2d1acc89dd06b5921126

**Pull request merged**